flyingsnake/__main__.py

```diff
@@ -22,6 +35,14 @@
           help="Draw the liquids present in the world.", default=None)
 @c.option("--paint/--no-paint", "draw_paint",
           help="Draw painted blocks with the paint color overlayed on them.", default=True)
+@c.option("-x", "--x_coord", "min_x",
+          help="Min x coord for custom world rendering", default=DEFAULT_COORDS)
```

Comment:
Instead of setting global constants, it is more pythonic to use `None` as a default.

A variable can be compared with `None` with the syntax:
```python
if variable is None:
  ...
```

```python
if variable is not None:
  ...
```